### ISSUE TITLE
[Clojure] Implement new Concept Exercise: basics

### ISSUE THREAD

**Issue by porkostomus** [exercism/v3#1508]
This issue describes how to implement the `basics` concept exercise for the Clojure track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of programming in Clojure.

## Learning objectives

- Know what a value is.
- Know that values are immutable.
- Know how to define a binding.
- Know how to define a function.
- Know how to return a value from a function.
- Know how to call a function.
- Know how to use mathematical operators on integers.
- Know how to define single- and multiline comments.

## Out of scope

- Reference types.
- Memory and performance characteristics.
- Higher-order functions.
- Recursion.
- Anonymous functions
- Function composition.
- Pure functions.
- Partial application.
- Organizing functions in namespaces.

## Concepts

The Concepts this exercise unlocks are:

- `basics`: - Know what a value is, know that values are immutable, know how to define a binding, know how to define a function, know how to return a value from a function, know how to call a function, know how to use mathematical operators on integers, know how to define single- and multiline comments.

## Prerequisites

There are no prerequisites.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

The Clojure [analyzer][analyzer] is currently being developed. Instructions to follow ;)

## Implementing

To implement a concept exercise, the following files must be created:

languages
└── clojure
    └── exercises
        └── concept
            └── <SLUG>
                ├── .docs
                |   ├── instructions.md
                |   ├── introduction.md
                |   ├── hints.md
                |   └── after.md (optional)
                ├── .meta
                |   |── design.md
                |   └── Example.clj
                ├── <NAME>.clj
                └── <NAME>_test.clj

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[analyzer]: https://github.com/exercism/clojure-analyzer
[representer]: https://github.com/exercism/clojure-representer

**Comment by porkostomus** [comment]
Done (see #1509)
